### DISCUSSION TITLE
Configure Dokan and pull requests ??

### DISCUSSION THREAD

**Discussion by pntrafewl** [kubilus1/autoortho#77] [General]
How do I configure Dokan so autoortho will find/use it? Also, do merged pull requests go into the latest release 0.3.0? I'm definitely no github expert, but I'm trying to learn.

**Comment by kubilus1** [comment]
The latest release should just detect you have Dokan installed now.  Prior releases required to you to enable the 'winfuse = True' switch in the config file, but that detection is now the default.

As for pull requests, it's up to how the project is structured.  My approach is to test and validate the PRs, and eventually merge to 'main'.  Then from time to time I'll make a point-in-time release from the main branch.
